pkg/operator/ceph/cluster/osd/osd_test.go

```diff
@@ -565,10 +565,10 @@ func TestDetectCrushLocation(t *testing.T) {
 
 	// update the location with valid topology labels
 	nodeLabels = map[string]string{
-		"failure-domain.beta.kubernetes.io/region": "region1",
-		"failure-domain.beta.kubernetes.io/zone":   "zone1",
-		"topology.rook.io/rack":                    "rack1",
-		"topology.rook.io/row":                     "row1",
+		"topology.kubernetes.io/region": "region1",
+		"topology.kubernetes.io/zone":   "zone1",
+		"topology.rook.io/rack":         "rack1",
+		"topology.rook.io/row":          "row1",
```

Comment:
We should leave these checks. Even though the label is deprecated, support is not fully removed. In K8s, "deprecated" is not synonymous with "unsupported." Kubernetes docs still include them in their well-known labels list: https://kubernetes.io/docs/reference/labels-annotations-taints/